reqs/requirements-dev-contrib.txt

```diff
@@ -3,4 +3,5 @@ black[jupyter]==23.3.0
 invoke>=2.0.0
 mypy==1.3.0
 pre-commit>=2.21.0
-ruff==0.0.271
\ No newline at end of file
+ruff==0.0.271
+tomli>=2.0.1
```

Comment:
Add `tomli` library as a dev dependency

Reply:
For reference: [tomli](https://pypi.org/project/tomli/) is a TOML parsing package